src/pages/en/install/manual.md

```diff
@@ -30,18 +32,47 @@ cd my-astro-project
 
 Once you are in your new directory, create your project `package.json` file. This is how you will manage your project dependencies, including Astro. If you aren't familiar with this file format, run the following command to create one.
 
-```bash
-npm init --yes
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm init --yes
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm init 
```

Comment:
I’m not entirely familiar with the `--yes` flag but shouldn’t it be included for pnpm like it is for npm & yarn?

Reply:
```suggestion
  pnpm init --yes
```

Reply:
`pnpm` decided to be "better" than `npm` by doing the "--yes" thing by default! (["`pnpm init` is non-interactive by default"](https://github.com/pnpm/pnpm/discussions/3536))